api/models.py

```diff
@@ -26,6 +26,9 @@
 
 logger = logging.getLogger(__name__)
 
+news_url = "https://api.cognitive.microsoft.com/bing/v7.0/news/search"
+search_url = "https://api.cognitive.microsoft.com/bing/v7.0/search"
```

Comment:
As these variables are supposed to be constants, we should name them in upper-case letters : `NEWS_URL` and `SEARCH_URL`